db/init/05-data-source.sql

```diff
@@ -3,6 +3,20 @@
 
 
 
+/*Create function to encrypt password column*/
+CREATE OR REPLACE FUNCTION base.encrypt_password()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW.password = PGP_SYM_ENCRYPT(NEW.password,'AES_KEY');
```

Comment:
Add a space after the comma: `(NEW.password, 'AES_KEY')`
Also: where's the `AES_KEY` coming from?

Reply:
I think we should have different keys for different scenarios though. Additionally the keys we already have are used for asymmetric encryption. This is symmetric encryption.

Reply:
To keep the implementation simple, I have done the following:
- create a new schema and table configuration.parameter
- generate secret key during database setup and store it in configuration.parameter
- update trigger and view to encrypt data using the key